modules/dnn/src/onnx/onnx_graph_simplifier.cpp

```diff
@@ -1704,30 +1706,84 @@ void simplifySubgraphs(opencv_onnx::GraphProto& net)
     simplifySubgraphs(Ptr<ImportGraphWrapper>(new ONNXGraphWrapper(net)), subgraphs);
 }
 
-Mat getMatFromTensor(const opencv_onnx::TensorProto& tensor_proto, bool uint8ToInt8)
+
+
+static char* getTensorRAWData(const opencv_onnx::TensorProto& tensor_proto,
+                              std::vector<int64_t>& tensor_data, const std::string& base_path = "")
+{
+    if (tensor_proto.has_data_location() && tensor_proto.data_location() == opencv_onnx::TensorProto::EXTERNAL) {
+    #if OPENCV_HAVE_FILESYSTEM_SUPPORT
+        CV_Assert(tensor_proto.has_data_location() && tensor_proto.data_location() == opencv_onnx::TensorProto::EXTERNAL);
+        auto it_begin = tensor_proto.external_data().begin();
+        auto it_end = tensor_proto.external_data().end();
+        // file path
+        auto it = std::find_if(it_begin, it_end,[](const auto& entry) { return entry.key() == "location"; });
+        CV_Assert(it != it_end);
```

Comment:
I propose to use CV_check with verbose error message. It's not useful for production usage, but may be practical for development.

Reply:
I have changed this to `CV_CheckTrue`